MERGE TIME!

flukeout opened the pull request:
**Powerups**
* Added some styles when the paddle has multiple powerups
  * For example, sticky + noclip gives the paddle a black border instead of white
  * Still need to do spin + sticky somehow
* All powerups now cause some particles to come out of the endzone when they are scored with
* Added a cool vertical beam when powerups score
* The spin powerup, when activated by hitting the side-wall, always goes towards the opponent's side of the court now

**Screen Fading**
* Splash fades to black
* Game fades in from black

**Cleanup**
* Some random CSS cleanup 
* chucked some unused images

**Scoring Percentages**
* Remove 'em in favor of "OW" etc.

flukeout commented:
Gonna add some more to this... don't merge yet plz

secretrobotron commented on battle-pong/splash.html:
```diff
@@ -23,22 +24,20 @@
-
       <div class="game-title">
-        <div class="actual-title">DEATHSTROID</div>
+        <div class="actual-title">SPACEBARS</div>
```
Man, I don't know about the name change. I think it's a bit like the rotation stuff: let's pick one and stick with it because it'll always be in question otherwise.

Want to do a little poll among friends to put the matter to bed? I can ask a few people today and you do the same?

flukeout commented:
Reverted game title and fixed the crazy slowdown during end of round phases. It had to do with adding a radial background on the body tag (for the purple or green sky). Moving that to the `.sky` div fixed it right up.

flukeout commented:
Adding rules between splash and game...

flukeout commented:
@secretrobotron I've got some content in /rules.html - check it out if you get a chance. Still WIP

secretrobotron commented:
Man, the rules are lookin' really slick. I added some functionality to move between the options using the keyboard arrow keys. I'd love to use a doubles' tennis metaphor, but I don't know any. Instead, I'll just talk about JavaScript: the code I wrote is one step away from being able to use the gamepad to move around the rules screen. Should be easy to connect now :).

secretrobotron commented:
@flukeout merge this into the branch you're working on: https://github.com/flukeout/games/pull/2

flukeout commented:
Hey cool merged your work and also added buttons to start the game and go back to splash and linked them to the keyboard. We'll need to add a way to go back to rules or splash from the game too i think.

secretrobotron commented:
@flukeout i think this is quite mergable. Any reason to keep it aside?

flukeout commented:
Yeah will merge it later tonight, just ironing a few things out.

flukeout commented:
Added some little hand icons for the selected button. We gotta enable this control style for the controllers too and on the splash / pause page.

![image](https://user-images.githubusercontent.com/25212/37882481-808af866-3058-11e8-95fc-0495fb042f13.png)

ryanwarsaw commented:
that looks siiiiiick man, good work

flukeout commented:
Oh hey @ryanwarsaw - thanks man! We're still working on this thing, but trying to finish soon haha.
